openoereb/pyramid_oereb
# Use custom params from url GEO-2777

This replaces #985 

Addresses https://jira.camptocamp.com/browse/GEO-2777
Adds a parameter to get all the config params from a URL and add them in the print request for that map.
Custom parameters need to be specified in a list in the config.

This is implemented for the base maps requested in the restriction on landownership

TO TEST:

- Add a parameter in the [config.yaml.make](https://github.com/openoereb/pyramid_oereb/pull/986/files#diff-9a53f2e0745156473e893a25c2999c0eR128-R130)
    - Ideally you add a parameter that is not already added standard wise and 
    - witch exists in the URL (i.e BBOX). 
- The parameter should sow in the getspec under `customParams` 
- If nothing is specified the the parameter `TRANSPARENT` will appear in the `customParams` as in master.
